cor_graphical/verification/verification_report.Rmd

```diff
@@ -0,0 +1,453 @@
+  ds1 = plot.25sample1_verification, ds2 = original.plot.25sample1
+)
+
+```
+```{r echo=FALSE}
+
+clean_comparison_pass <- all(sapply(data_clean_comparison, `[[`, "equal"))
+
+```
+
+`plot.25sample1` generated by using `cor_graphical_verification_code.R` was `r ifelse(clean_comparison_pass, "equivalent to","different from")` the output generated by original programmer, and `r ifelse(clean_comparison_pass, "passes","fails")` verification.
+
+### Verification of lineplot of Binding Antibody to Spike: baseline negative vaccine arm (2 timepoints)
+
+```{r, include=FALSE}
+boxplots_neg_vaccine_bindSpike_2_verification <- read.csv(
+  here("verification/output/dat.longer.cor.subset.plot1_verification.csv")) %>%   
+  filter(Trt == "Vaccine", 
+         Bserostatus == "Baseline Neg", 
+         assay == "bindSpike",
+         time %in% c("Day 29","Day 57")) %>% 
+  arrange(Ptid,assay,time)
+
+original.boxplots_neg_vaccine_bindSpike_2<-read.csv(
+  here("verification/input/longer_cor_data_plot1.csv")) %>%
+  filter(Trt == "Vaccine", 
+         Bserostatus == "Baseline Neg", 
+         assay == "bindSpike",
+         time %in% c("Day 29","Day 57")) %>% 
+  arrange(Ptid,assay,time)
+
+plot.25sample1<-read.csv(
+  here("verification/input/plot.25sample1.csv")
+)
+
+lineplots_neg_vaccine_bindSpike_plot.25sample1_2_verification <- plot.25sample1_verification %>%
+  filter(Trt == "Vaccine", 
+         Bserostatus == "Baseline Neg", 
+         assay == "bindSpike",
+         time %in% c("Day 29","Day 57"))
+
+original.lineplots_neg_vaccine_bindSpike_plot.25sample1_2 <- plot.25sample1 %>%
+  filter(Trt == "Vaccine", 
+         Bserostatus == "Baseline Neg", 
+         assay == "bindSpike",
+         time %in% c("Day 29","Day 57"))
+
+
+```
+
+
+
+```{r comparison3}
+data_clean_comparison_boxplot <- compare_datasets(
+  cols =  colnames(boxplots_neg_vaccine_bindSpike_2_verification), index = "Ptid",
+  ds1 = boxplots_neg_vaccine_bindSpike_2_verification, 
+  ds2 = original.boxplots_neg_vaccine_bindSpike_2
+)
+
+data_clean_comparison_lineplot <- compare_datasets(
+  cols =  colnames(lineplots_neg_vaccine_bindSpike_plot.25sample1_2_verification), index = "Ptid",
+  ds1 = lineplots_neg_vaccine_bindSpike_plot.25sample1_2_verification, 
+  ds2 = original.lineplots_neg_vaccine_bindSpike_plot.25sample1_2
+)
+
+
+```
+```{r echo=FALSE}
+
+clean_comparison_pass_boxplot <- all(sapply(data_clean_comparison_boxplot, `[[`, "equal"))
+
+clean_comparison_pass_lineplot <- all(sapply(data_clean_comparison_lineplot, `[[`, "equal"))
+
+```
+
+The violin plot, box plot and line plot generated by the tester were visually compared with the outputs generated by original programmer. 
+
+Key statistics(min, 25th percentile, median, 75th percentile, max) shown in the box plot calculated by tester were `r ifelse(clean_comparison_pass, "equivalent to","different from")` the numbers provided by original programmer, and `r ifelse(clean_comparison_pass, "passes","fails")` verification.
```

Comment:
Uses wrong object for comparison - update to `clean_comparison_pass_boxplot`

Reply:
Bug has been fixed.